Lib/tkinter/ttk.py

```diff
@@ -1543,11 +1543,12 @@ def destroy(self):
         try:
             self._variable.trace_vdelete('w', self.__tracecb)
         except AttributeError:
-            # widget has been destroyed already
             pass
         else:
             del self._variable
-            Frame.destroy(self)
+        super().destroy()
+        self.label = None
```

Comment:
I dislike when attributes "disappear". I prefer setting them to None. Moreover, "self.label = None" doesn't fail if destroy() is called twice, whereas "del self._variable" requires a try/except AttributeError.